lib/Dialect/PTensor/Transforms/PTensorDist.cpp

```diff
@@ -48,32 +54,29 @@ namespace {
 // ***** Some helper functions ***
 // *******************************
 
-// create ::imex::dist::RegisterPTensorOp
-inline ::mlir::Value createInitWithRT(::mlir::Location &loc,
-                                      ::mlir::OpBuilder &builder, uint64_t rank,
-                                      ::mlir::Value gshape) {
-  return builder.create<::imex::dist::RegisterPTensorOp>(
-      loc, builder.getI64Type(), gshape);
+// create ::imex::dist::DistInfoOp
+inline ::mlir::Value createDistInfo(::mlir::Location &loc,
+                                    ::mlir::OpBuilder &builder, uint64_t rank,
+                                    ::mlir::Value gshape, ::mlir::Value team) {
```

Comment:
I am assuming `gshape` means shape of a group, but it's not very clear what do we exactly mean by that.

Reply:
see above, it's the global shape of the tensor (in contrast to the process-local shape).